2023.md

```diff
@@ -105,11 +105,12 @@
 * [PostgreSQL With JSON Data Types](https://medium.com/@wanshikawar.99/postgresql-with-json-data-types-c4d467b3c473) by [Rajitha Wanshika](https://medium.com/@wanshikawar.99)
 * [Vulnerability Management Process: It’s More Than Just Detecting Vulnerabilities in Software!](https://medium.com/@dinukathathsara/vulnerability-management-process-its-more-than-just-detecting-vulnerabilities-in-software-55f010f473b0) by [Dinuka Thathsara](https://medium.com/@dinukathathsara)
 
-## July (07)
+## July (08)
 * [Connect AWS Instance as Jenkins Slave Node](https://sachinmamoru.medium.com/connect-aws-instance-as-jenkins-slave-node-19fc9cc9a4ad) by [Sachin Mamoru](https://medium.com/@sachinmamoru)
 * [Optimizing AWS Resource Performance: A Comprehensive Guide to EC2, RDS, and CloudWatch Monitoring](https://sachinmamoru.medium.com/optimizing-aws-resource-performance-a-comprehensive-guide-to-ec2-rds-and-cloudwatch-monitoring-3cf8defc6435) by [Sachin Mamoru](https://medium.com/@sachinmamoru)
 * [Define REST APIs using AsyncAPI Specification](https://medium.com/@santhoshziyam/define-rest-apis-using-asyncapi-specification-a4e779ee58bf) by [Ziyam Santhosh](https://medium.com/@santhoshziyam)
 * [Register users with non-email alphanumeric usernames in Asgardeo organizations](https://medium.com/@dasunin30/register-users-with-non-email-alphanumeric-usernames-in-asgardeo-organizations-8b6a5e2fe6df) by [Dasuni Geeganage](https://medium.com/@dasunin30)
 * [Secure Your APIs with Asgardeo: A Step-by-Step Guide - Part 1](https://achinthaisuru444.medium.com/secure-your-apis-with-asgardeo-a-step-by-step-guide-part-1-799df090ab0e) by [Achintha Isuru](https://achinthaisuru444.medium.com/)
 * [Authenticate Next.js applications with Asgardeo-Part 1](https://medium.com/@omalwijegunawardana/authenticate-next-js-applications-with-asgardeo-part-1-eb02fbcf7fb9) by [Omal Wijegunawardana](https://medium.com/@omalwijegunawardana)
 * [Authenticate Next.js applications with Asgardeo-Part 2](https://medium.com/@omalwijegunawardana/authenticate-next-js-applications-with-asgardeo-part-2-2db5a45ebcb7) by [Omal Wijegunawardana](https://medium.com/@omalwijegunawardana)
+* [Secure Your APIs with Asgardeo: A Step-by-Step Guide - Part 2](https://achinthaisuru444.medium.com/secure-your-apis-with-asgardeo-a-step-by-step-guide-part-2-f5d90c1a6924) by [Achintha Isuru](https://achinthaisuru444.medium.com/)
```

Comment:
This is August though :D

Reply:
Fixed https://github.com/wso2/iam-blogs/pull/296/commits/e34f56a968cd27f30c3ee93bc70853ed14c07af2